### PR TITLE
Rewriting 2.5 and 2.6

### PR DESCRIPTION
**Pull request by andreas-junghanns**
No known API or functional changes added - please check.

### CODE REVIEW

File: docs/2_6_versioning_layered_standards.adoc [hunk 1]
```diff
@@ -2,19 +2,26 @@
...
-Layered standards that have achieved enough adoption or importance to be included into the base standard set could be incorporated into a new minor or major release version of the base standard as an optional or mandatory appendix, making support for this layered standard optional or required for conformance with the newly published minor release version of the base standard.
+Any FMU following a layered standard on top of FMI 3.0 must still be a compliant FMI 3.0 FMU.
+
+Layered standards can fall into one of three categories:
+
+* defined by third parties, completely independent from the <<MAP>> FMI,
+* defined by third parties that are endorsed by the FMI project and listed on the FMI project website, and
+* defined/adopted and published by the <<MAP>> FMI project itself, making them FMI project layered standards.
+
+Layered standards that have achieved enough adoption or importance to be included into the base standard could be incorporated into a new minor or major release version of the base standard as an optional or mandatory appendix, making support for this layered standard optional or required for conformance with the newly published release of the FMI standard.
```
Comment (chrbertsch) [inline docs/2_6_versioning_layered_standards.adoc:27]: There is also the possibility that we move content of a layered standard in the main text of a future minor or majorr release. Is this clear or do we have to state this explicitely?

File: docs/4___co-simulation.adoc [hunk 2]
```diff
@@ -4,7 +4,7 @@ FMI for Co-Simulation provides a standardized interface for execution of simulat
 In contrast to FMI for Model Exchange, a Co-Simulation FMU implements not only the model algorithm, but also the required solution method.
 The data exchange between FMUs is restricted to discrete communication points latexmath:[\mathbf{t}_i] (unless <<IntermediateUpdateMode>> is used).
 In the time between two communication points (or between entering <<IntermediateUpdateMode>>), the subsystem inside an FMU is solved independently by internal means.
-This leads to a delay for information "traveling through" the FMU, so there cannot be an immediate reaction (direct feedthrough) as in <<InitializationMode>>, <<EventMode>> or <<ContinuousTimeMode>>.
+This leads to a delay for information "traveling through" the FMU, so there cannot be an immediate reaction (direct feedthrough) as in <<InitializationMode>>, <<EventMode>> or <<ContinuousTimeMode>> of Model Exchange.
```
Comment (chrbertsch) [inline docs/4___co-simulation.adoc:7]: This is not the case in event mode, right? Should we state this explicitely?

File: docs/2_6_versioning_layered_standards.adoc (hunk 1 above)
Comment (andreas-junghanns) [reply docs/2_6_versioning_layered_standards.adoc:27]: Isn´t that what line 27 says?

File: docs/4___co-simulation.adoc (hunk 2 above)
Comment (andreas-junghanns) [reply docs/4___co-simulation.adoc:7]: Line 7 lists the modes without delay capabilities. I am not sure what you are missing here...

File: docs/2_6_versioning_layered_standards.adoc (hunk 1 above)
Comment (chrbertsch) [reply docs/2_6_versioning_layered_standards.adoc:27]: It only talks about an appendix, not about the main standard text

File: docs/4___co-simulation.adoc (hunk 2 above)
Comment (chrbertsch) [reply docs/4___co-simulation.adoc:7]: You are right. Please ignore my comment.

File: docs/2_6_versioning_layered_standards.adoc (hunk 1 above)
Comment (andreas-junghanns) [reply docs/2_6_versioning_layered_standards.adoc]: I see, fixed.

**Review by chrbertsch — APPROVED** [review]

**Pull request merged**